lib/pgcrypto/manipulation.rb

```diff
@@ -12,12 +12,16 @@ def process_arel( arel, binds = [] )
       when Arel::UpdateManager
         process_update(arel, binds)
       end
+
       return arel, binds
     end
 
-    def process_raw_relation( relation ) # For use where AR builds a relation without a surrounding statement.
-      r = translate_where( relation )
-      Array === r ? r.inject{ |x,y| x.and y } : r
+    # For use where AR builds a relation without a surrounding statement.
+    def process_raw_relation(relation)
+      # r = translate_where(relation.where_clause.ast)
+      # r.is_a?(Array) ? r.inject { |x, y| x.and y } : r
+      # relation.where_clause = ActiveRecord::Relation::WhereClause.new(testr)
+      relation
```

Comment:
If this method is going to be a no-op, should we just remove it entirely instead?